Bio/PDB/MMCIF2Dict.py

```diff
@@ -31,6 +31,10 @@ def __init__(self, filename):
             except StopIteration:
                 return  # for Python 3.7 and PEP 479
             self[token[0:5]] = token[5:]
+            if not token[0:5].startswith("data_"):
+                raise ValueError(
+                    f"The mmCIF file {filename} does not start with 'data_'"
```

Comment:
I'd write: "The input mmCIF file must begin with a 'data_' directive."

Reply:
Absolutely. I'll do that hopefully either today or tomorrow.

Reply:
A point on the error message, however, it may be better to have something like "The input mmCIF file {filename} must begin with a 'data_' directive.", as the filename may make debugging a little easier.